<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QPushButton" name="login_pushbutton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 220, 220);</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome_text_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>231</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>601</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signup_pushbutton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="password_layout" stretch="1,2">
    <item>
     <widget class="QLabel" name="password_label">
      <property name="font">
       <font>
        <family>..黑体UI-日本语</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password_line_edit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="username_layout" stretch="1,2">
    <item>
     <widget class="QLabel" name="user_label">
      <property name="font">
       <font>
        <family>..黑体UI-日本语</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_line_edit">
      <property name="font">
       <font>
        <family>.黑体-日本语</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="config_pushbutton">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>390</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
